extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -37,6 +37,13 @@
     "pageview": "https://hm.baidu.com/hm.gif?si=!token&nv=0&st=4&v=pixel-1.0&rnd=_timestamp_&et=0",
     "event": "https://hm.baidu.com/hm.gif?si=!token&nv=0&st=4&v=pixel-1.0&rnd=_timestamp_&ep=!category*!action*!label*!value&et=4&api=8_0"
   },
+  "blueconic": {
+    "tenantId": "_tenantId_",
+    "pixelId": "_pixelId_",
+    "property": "_property_",
+    "extraTrackingValues": "&customTitle=_title_",
```

Comment:
Remove these as they are not requests defined in your .json

Reply:
Are removed